packages/core/admin/admin/src/content-manager/components/DynamicZone/components/Component/hooks/useMainValue.js

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
+import { get, toString } from 'lodash';
```

Comment:
Could you change these imports to be:

```suggestion
import get from 'lodash/get';
import toString from 'lodash/toString';
```

as this tree-shakes better?